### ISSUE TITLE
Quantization fails using ARMNN if layers are in particular Sequence

### ISSUE THREAD

**Issue by uu-praveeng** [ARM-software/armnn#328]
Hi,

I have created ResNet FP32 model using armnn APIs. When I tried to quantize it fails with below error 

```
Thread 1 "ArmnnQuantizer" received signal SIGSEGV, Segmentation fault.
0x0000007fbe7a7454 in armnn::OutputSlot::CalculateIndexOnOwner() const () from ./libarmnn.so.19
```

But when I change the order of layers like moving the poolLayer1 below convLayer10. It proceeds and gives out error simialr to issue https://github.com/ARM-software/armnn/issues/321 

Attached is ResNet.cpp(contains portion of ResNet model)  which is used for creation of FP32 model and then used for quantization.
Let me know if you need any further info.

[ResNetQuery.zip](https://github.com/ARM-software/armnn/files/4058036/ResNetQuery.zip)

Thank you,
Praveen.

**Comment by MikeJKelly** [comment]
Hi Praveen, 

I'm taking a look at this bug now and I hope to have some information for you soon.

**Comment by uu-praveeng** [comment]
Ok @MikeJKelly . Let me know if you need any inputs . 

Thank you,
Praveen

**Comment by MikeJKelly** [comment]
With the fix for #321 I cannot reproduce this issue (I'm getting a separate unrelated issue around quantized scales) can you confirm that you no longer have to rearrange the order of layers to avoid the segfault?

**Comment by MatthewARM** [comment]
Closing this per comment acknowledging fix on #321
